metabase/metabase
# rename jest config file to default naming to make it pickup automatic…

### Description

This renames the jest configuration file to the default naming, so that it's picked up automatically by jest and related tools.

This would allow us for example to use, out of the box, some vscode extensions that could help in development such as:
- https://marketplace.visualstudio.com/items?itemName=Orta.vscode-jest (this adds the green checks and runs tests automatically in watch mode)
- https://marketplace.visualstudio.com/items?itemName=firsttris.vscode-jest-runner (this adds the "run" and "debug" actions on tests, which are very useful when working/debugging specific tests)

<img width="827" alt="image" src="https://github.com/metabase/metabase/assets/1914270/da44da6c-b66b-42be-ad0f-7760c4467d87">


### How to verify

If the CI is green, everything should be fine.
We can manually run jest tests to check if they pass.
